### PR TITLE
added shubham.json

### PR DESCRIPTION
**Pull request by neverwannafly**
Added a new name file.

### CODE REVIEW

**Comment by neverwannafly** [comment]
I changed {"shubh"} to ["shubh"]. Shall I commit the changes to my forked branch and then create a new pull request ?

File: collection/shubham.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+	"name" : "shubham",
+	"meaning" : "auspicious, lucky",
+	"aliases" : ["shubh"],
+	"translations" : {
+		"jap" : "シュハム",
```
Comment (arminkhoshbin) [inline collection/shubham.json]: Can you please update this to use the correct ISO code? The correct ISO code for Japanese is `jpn`.

**Comment by arminkhoshbin** [comment]
Hey @neverwannafly,
No extra Pull request is needed. You can push all your changes to your fork repository.
Can you please address the requested change? This should fix the failing test and then we can merge this PR for you :)

Thanks!

**Comment by neverwannafly** [comment]
I am terribly sorry ! 
I have just started exploring the world of OS :'D !
Did the required changes !
Thanks for guiding me along ^_^

**Comment by CodeWritingCow** [comment]
@neverwannafly you are welcome. Thanks for contributing!

**Pull request merged**